mephisto/scripts/local_db/gh_actions/auto_generate_provider.py

```diff
@@ -0,0 +1,32 @@
+from mdutils.mdutils import MdUtils
+from mephisto.client.cli_commands import get_wut_arguments
+from mephisto.operations.registry import (
+    get_valid_provider_types,
+)
+from mephisto.scripts.local_db.gh_actions.auto_generate_blueprint import (
+    create_blueprint_info,
+)
+
+
+def main():
+    provider_file = MdUtils(
+        file_name="../../../../docs/web/docs/reference/providers.md",
+    )
+    provider_file.new_header(level=1, title="Provider")
+    provider_file.new_paragraph(
+        "The crowd provider determines the source of the crowd workers."
```

Comment:
I think I would borrow from https://mephisto.ai/docs/explanations/architecture_overview/#crowdproviders to say something like:

> Crowd providers standardize access to external crowd workers, by wrapping external API communication through a standardized interface.